CMakeLists.txt

```diff
@@ -2141,22 +2141,53 @@ endif()
 
 
 if(APPLE)
-  find_library(COREFOUNDATION_LIBRARY CoreFoundation REQUIRED)
-  target_link_libraries(mixxx-lib PRIVATE ${COREFOUNDATION_LIBRARY})
+  if(QT5_TYPE STREQUAL "STATIC_LIBRARY")
+    find_package(FFTW REQUIRED)
+    find_package(SndFile REQUIRED)
+    find_package(sord CONFIG REQUIRED)
+    find_package(SQLite3 REQUIRED)
+    find_library(SAMPLERATE_LIBRARY samplerate REQUIRED)
+    target_link_libraries(mixxx-lib PRIVATE
+        FFTW::FFTW
+        SndFile::sndfile
+        sord::sord
+        ${SQLite3_LIBRARIES}
+        ${SAMPLERATE_LIBRARY}
+    )
```

Comment:
Aren't these handled elsewhere in CMakeLists.txt?

Reply:
I have tried to get rid of some of them but these are the remaining bits.